pragma-node/src/handlers/entries/get_onchain/ohlc.rs

```diff
@@ -1,2 +1,101 @@
-// TODO(akhercha): ohldc endpoint
-// https://buildonpragma.notion.site/Pragma-API-fc14ba680030470cab61ee58098b135f
+use std::time::Duration;
+
+use axum::extract::{Query, State};
+use axum::response::IntoResponse;
+use serde_json::json;
+
+use pragma_common::types::{Interval, Network};
+
+use crate::handlers::entries::utils::currency_pair_to_pair_id;
+use crate::handlers::entries::GetOnchainOHLCParams;
+use crate::infra::repositories::entry_repository::OHLCEntry;
+use crate::infra::repositories::onchain_repository::get_ohlc;
+use crate::utils::PathExtractor;
+use crate::AppState;
+
+use axum::extract::ws::{Message, WebSocket, WebSocketUpgrade};
+
+pub const WS_UPDATING_INTERVAL_IN_SECONDS: u64 = 10;
```

Comment:
`WS_UPDATE_INTERVAL_IN_SECONDS` might be a better name. 
Also how low can this go, is 10ms possible ? I imagine that the ohlc computation is quite fast when just sending them one by one. There are 2 use cases here:
1. The subscriber is listening permanently to data updates and is ok with having duplicates
2. The subscriber only wants updates when the value changes

Reply:
I think we can only focus on case 2 for now